pkg/rc/replication_controller.go

```diff
@@ -42,7 +43,13 @@ func New(
 	rcStore rcstore.Store,
 	scheduler Scheduler,
 	podApplicator labels.Applicator,
```

Comment:
@petertseng you could also do something like this to split up PodApplicator and RCApplicator interfaces: https://github.com/kubernetes/kubernetes/blob/master/pkg/runtime/interfaces.go#L144